Config/SelfCheckingAsseticResource.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony framework.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\Bundle\AsseticBundle\Config;
+
+use Symfony\Component\Config\Resource\SelfCheckingResourceInterface;
+
+/**
+ * Implmenets SelfCheckingResourceInterface required in Symfony 3.0.
+ *
+ * @author Ryan Weaver <ryan@knpuniversity.com>
+ */
```

Comment:
please mark the class as ``@internal``. I don't want people to start using it themselves